translations/ko/LC_MESSAGES/messages.po

```diff
@@ -88,27 +88,27 @@ msgstr "AMP 페이지"
 
 #: /content/includes/menu.yaml
 msgid "AMP Playground"
-msgstr ""
+msgstr "AMP 플레이그라운드"
 
 #: /content/includes/lists.yaml
 msgid "AMP Roadmap"
 msgstr "AMP 로드맵"
 
 #: /views/partials/promo_banner.html:18
 msgid "AMP Roadshows."
-msgstr ""
+msgstr "AMP 로드쇼."
 
 #: /content/includes/menu.yaml
 msgid "AMP Start"
-msgstr ""
```

Comment:
AMP Start

AMP Start is a products name. So it is better non-translation than translation.